docs/src/modules/java/pages/quickstart/cr-value-entity-java-kafka.adoc

```diff
@@ -0,0 +1,290 @@
+= Quickstart: Customer Registry in Java
+
+include::ROOT:partial$include.adoc[]
+include::java:partial$attributes.adoc[]
+
+Learn how to create a customer registry in Java, package it into a container, and run it on Akka Serverless.
+
+== Before you begin
+
+* If you're new to Akka Serverless, {console}[create an account, window="console"] so you can try out Akka Serverless for free.
+* You'll also need to install the https://developer.lightbend.com/docs/akka-serverless/akkasls/install-akkasls.html[Akka Serverless CLI, window="new-doc"] if you want to deploy from a terminal window.
+* For this quickstart, you'll also need
+** https://docs.docker.com/engine/install[Docker {minimum_docker_version} or higher, window="new"]
+** Java {java-version} or higher
+** https://maven.apache.org/download.cgi[Maven 3.x or higher, window="new"]
+** https://github.com/fullstorydev/grpcurl#installation[`grpcurl`, window="new"]
+
+[NOTE]
+====
+If you want to bypass writing code and jump straight to the deployment:
+
+. Download the source code using the Akka Serverless CLI: 
+`akkasls quickstart download customer-registry-java`
+
+. Skip to <<Package and deploy your service>>.
+====
+
+
+== Writing the Customer Registry
```

Comment:
I know already touched on this in the issue but want to make sure there is no misunderstanding. It should be similar to the Customer Registry **with Views** quickstart. It's referring to the first quickstart in https://github.com/lightbend/akkaserverless-java-sdk/blob/main/docs/src/modules/java/pages/quickstart/cr-value-entity-views-java.adoc#start-from-the-customer-registry-entity

In that way this guide can be focused only on the "add Kafka" part.

Reply:
I see. Ok. Thanks for the heads up. I'll follow the same pattern as the customer registry with views .adoc